pkg/sql/colexec/execgen/testdata/template

```diff
@@ -0,0 +1,142 @@
+inline
+package main
```

Comment:
I think it would be useful to have the template test run without the inliner steps